code/modules/cargo/supplypod.dm

```diff
@@ -41,7 +41,7 @@
 /obj/structure/closet/supplypod/bluespacepod
 	style = STYLE_BLUESPACE
 	bluespace = TRUE
-	explosionSize = list(0,0,1,2)
+	explosionSize = list(0,0,0,0) //So we dont kill mobs or brake bombs 
```

Comment:
This is just hugboxing, and doesn't actually stop the pod from being lethal. The only thing this will do is make it safe to be in the immediate area around the pod when it lands.

Reply:
When a drop pod lands it explodes and hits all items in the tile even itself, and the crate it has. If you get Crab Rocket a Cortex Commander joke, then all mobs or crabs will be killed in the rocket as soon as it lands, this goes with any animal that is small or peaceful, cows/hens/ect will die or be at a deaths door. Also this tents to set of smoke bombs if its in the crate as well, *same for flashes and such*